
{'date': 'Fri Apr 17 2020 18:33:27.330 GMt-0300 (GMT-03:00) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 3
    from _spy.vitollino.main import Inventaio as inv
ImportError: cannot import name 'Inventaio'

ImportError
Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 264
    action()
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 3
    from _spy.vitollino.main import Inventaio as inv
'''},
{'date': 'Fri Apr 17 2020 18:46:53.408 GMt-0300 (GMT-03:00) -X- SuPyGirls -X-',
'error': '''
 module <string> line 6
  PLAYER = 
            ^
SyntaxError: invalid syntax
'''},
{'date': 'Fri Feb 11 2022 15:33:00.311 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 22
    Reticulo()
  module <module> line 10
    self.vai_reticulo()
  module <module> line 12
    sala_a = Sala(*[IM.format(lnk) for lnk in RETICULO[0]])
NameError: name 'Sala' is not defined
'''},
{'date': 'Fri Feb 11 2022 15:50:35.505 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 24
    Reticulo()
  module <module> line 11
    self.vai_reticulo()
  module <module> line 19
    Elemrnto("https://i.imgur.com/cTgMqWX.png", cena =sala_b.norte)
NameError: name 'Elemrnto' is not defined
'''},
{'date': 'Fri Feb 11 2022 16:16:19.959 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 180
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 310
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 282
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 299
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 24
    Reticulo()
  module <module> line 11
    self.vai_reticulo()
  module <module> line 19
    Elemrnto("https://i.imgur.com/cTgMqWX.png", cena =sala_b.norte)
NameError: name 'Elemrnto' is not defined
'''},
{'date': 'Fri Feb 11 2022 16:27:25.941 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 5
  MARIA = "https://i.imgur.com/FukdPW2.jpg""
                                            ^
SyntaxError: EOL while scanning string literal
'''},
{'date': 'Fri Feb 11 2022 17:29:12.340 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 61
  Personagem(MARIA, tit="MARIA", x= 700, y=200, w=200, h=300, cena =sala_a.norte
                                                                                          ^
SyntaxError: invalid syntax
'''},